CHANGELOG.md

```diff
@@ -0,0 +1,81 @@
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+### Changed
+### Removed
+
+## 0.16.0
+
+### Added
+- Improve AWS subcommands in ore, support AWS Pro downloading in cork, adjust LTS handling in plume ([#152](https://github.com/kinvolk/mantle/pull/152))
+- kola: Add a filter to run tests based on offering ([#158](https://github.com/kinvolk/mantle/pull/158))
+- cmd/cork: Allow to apply a patch on top of the manifest references ([#163](https://github.com/kinvolk/mantle/pull/163))
+- kola: provide internet access to qemu VM ([#167](https://github.com/kinvolk/mantle/pull/167))
+- platform: allow CLC templating for dynamic IP address insertion ([#168](https://github.com/kinvolk/mantle/pull/168))
+- kola: add kubeadm tests ([#171](https://github.com/kinvolk/mantle/pull/171))
+- kola/docker: add selinux test ([#177](https://github.com/kinvolk/mantle/pull/177))
+- kola/kubeadm: test various CNIs ([#182](https://github.com/kinvolk/mantle/pull/182))
+- kola/docker: accept 'cgroupns' security option ([#188](https://github.com/kinvolk/mantle/pull/188))
+- kola/kubeadm: add kubernetes 1.22 test ([#196](https://github.com/kinvolk/mantle/pull/196))
+- kola: support nightly version in version comparisons ([#198](https://github.com/kinvolk/mantle/pull/198))
+- kola/harness: detect LTS major version ([#200](https://github.com/kinvolk/mantle/pull/200))
+- Add platform for external provisioning ([#212](https://github.com/kinvolk/mantle/pull/212))
+- update Azure SDK for Gen2 VM support ([#214](https://github.com/kinvolk/mantle/pull/214))
+
+### Changed
+- kola/tests/misc/network.go: Allow systemd-resolved to run ([#153](https://github.com/kinvolk/mantle/pull/153))
+- kola/tests/misc/network.go: Disallow the CRI plugin to listen on TCP ([#154](https://github.com/kinvolk/mantle/pull/154))
+- kola/tests/misc/network.go: Allow the containerd CRI plugin to listen ([#155](https://github.com/kinvolk/mantle/pull/155))
+- cmd/cork/downloadimage: Make check for version.txt optional ([#156](https://github.com/kinvolk/mantle/pull/156))
+- cmd/plume: Don't try to publish GCE LTS images ([#157](https://github.com/kinvolk/mantle/pull/157))
+- kola: fix cgroup parameters for docker ([#29](https://github.com/kinvolk/mantle/pull/29))
+- Update git URLs to kinvolk org ([#159](https://github.com/kinvolk/mantle/pull/159))
+- kola/tests/flannel: use docker0's interface address as destination ([#161](https://github.com/kinvolk/mantle/pull/161))
+- Update Kubernetes test ([#162](https://github.com/kinvolk/mantle/pull/162))
+- cork: Apply patches with a dummy committer ([#164](https://github.com/kinvolk/mantle/pull/164))
+- kola/readme: update example to use container linux ([#166](https://github.com/kinvolk/mantle/pull/166))
+- Update golang.org/x/crypto and golang.org/x/net ([#173](https://github.com/kinvolk/mantle/pull/173))
```

Comment:
Maybe we can put this kind of changed under `### Security` section ?

Reply:
Good point. We can move this to `###Security`.